taskboard/interface.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict, Any
+
+class TaskBoard(ABC):
+    @abstractmethod
+    def get_tasks(self, list_id: str) -> List[Dict[str, Any]]:
+        """Return list of task dicts for a given list ID."""
```

Comment:
Can we create a task type that is a dataclass or something?
Instead of using raw dicts for it?

Reply:
Created a dataclass for Task and TaskList types.